starwars/src/components/Card.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import '../App.css';
+import App from '../App.js'
+
+
+const Card = props => {
+    console.log(props.data[1].name);
```

Comment:
Try just looking at `props.data` then mapping over it to see each specific object. My guess you're having an issue because React is attempting to do this before the items are in data.